opm/simulators/linalg/MILU.hpp

```diff
@@ -25,6 +25,31 @@
 #include <string>
 #include <vector>
 
+
+
+class auxiliaryFunctors
+{
+public:
+  static std::function<double(const double&)> IdentityFunctor(); 
+  static std::function<double(const double&)> OneFunctor(); 
+  static std::function<double(const double&)> SignFunctor(); 
+  static std::function<double(const double&)> IsPositiveFunctor(); 
+  static std::function<double(const double&)> AbsFunctor(); 
+};
+
+//std::function<double(const double&)>
+static std::function<double(const double&)> auxiliaryFunctors::IdentityFunctor(const double& t){return t;};
```

Comment:
You are returning a double and not a function. Maybe return the appropriate lambda expresssion?

Reply:
Too optimistic! :/